client/settings/test/settings-section.tsx

```diff
@@ -33,7 +33,7 @@ describe( 'SettingsSection', () => {
 
 	test( 'renders children', () => {
 		render(
-			<SettingsSection title="Foo" description="Bar">
```

Comment:
Removing this since `description` cannot be of `string` type

Reply:
Got it, thanks!